psydac/feec/old/__init__.py

```diff
@@ -0,0 +1,4 @@
+__all__ = ['utilities','derivatives']
```

Comment:
Why do we need an **old** directory inside **psydac/feec**?

Reply:
it will be deleted in the futur when we are sure that everything that was implemented in the old folder in updated in the new one